phpBB/assets/javascript/installer.js

```diff
@@ -0,0 +1,221 @@
+/**
+ * Installer's AJAX frontend handler
+ */
+
+(function($) { // Avoid conflicts with other libraries
+    // Global variables
+    var pollTimer = null;
+    var nextReadPosition = 0;
+
+    // Template related variables
+    var contentWrapper = $('.install-body').find('.main');
+
+    // Intercept form submits
+    intercept_form_submit($('#install_install'));
+
+    function poll_content(xhReq)
+    {
+        var messages = xhReq.responseText;
+
+        do {
+            var unprocessed = messages.substring(nextReadPosition);
+            var messageEndIndex = unprocessed.indexOf("}\n\n");
+
+            if (messageEndIndex != -1)
+            {
+                var endOfMessageIndex = messageEndIndex + 3; // 3 is the length of "}\n\n"
+                var message = unprocessed.substring(0, endOfMessageIndex);
+                parse_message(message);
+                nextReadPosition += endOfMessageIndex;
+            }
+        } while (messageEndIndex != -1);
```

Comment:
You should use exact equality operators, you've done it in other places too. `!==` not `!=`